### DISCUSSION TITLE
hw1 - policy on printing booleans

### DISCUSSION THREAD

**Discussion by sznoo** [prosyslab-classroom/cs424-program-reasoning#405] [Q&A] [answered]
### Your Name

서진우

### Question

hw1에서 print()가 boolean값도 출력 가능하게 구현해야 할까요? 메뉴얼의 문법에 따르면 가능해야 할 텐데 주어진 skeleton code는 int만 출력하는 것을 가정하는 것 같아 질문 드립니다.

Should I implement hw1's print() to support boolean type input? The instruction's semantic rules suggest it's true, but it seems only printing integers is considered by the skeleton code.

**Comment by jidnjs** [comment, marked as answer]
I think you can refer answer to question #289 .

**Reply by sznoo** [reply]
Thank you for your reply!

**Comment by spearo2** [comment]
Thank you @jidnjs for the quick response. Yes, #289 pretty much answers your question.

**Reply by sznoo** [reply]
Yes I had a clear answer. Thank you for your response:)
